perun-utils/ldapc-scripts/schemas/perun-schema.ldif

```diff
@@ -31,12 +31,19 @@ olcAttributeTypes: {26}( 1.3.6.1.4.1.8057.2.80.30 NAME 'loa' DESC 'Level of auth
 olcAttributeTypes: {27}( 1.3.6.1.4.1.8057.2.80.31 NAME 'internalUserIdentifiers' DESC 'All identifiers collected from UserExtSource attribute additionalIdentifiers' EQUALITY caseExactMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
 olcAttributeTypes: {28}( 1.3.6.1.4.1.8057.2.80.32 NAME 'perunFacilityDn' DESC 'DN of Facility the resource belongs to' EQUALITY distinguishedNameMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.12 SINGLE-VALUE )
 olcAttributeTypes: {29}( 1.3.6.1.4.1.8057.2.80.35 NAME 'eduPersonOrcid' DESC 'All OrcID identifiers of user' EQUALITY caseExactMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
+olcAttributeTypes: {30}( 1.3.6.1.4.1.8057.2.80.36 NAME 'checkGroupMembership' DESC 'OIDC check group membership' EQUALITY booleanMatch SYNTAX  1.3.6.1.4.1.1466.115.121.1.7 )
+olcAttributeTypes: {31}( 1.3.6.1.4.1.8057.2.80.37 NAME 'voShortNames' DESC 'Short names of assigned VOs' EQUALITY caseExactMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
+olcAttributeTypes: {32}( 1.3.6.1.4.1.8057.2.80.38 NAME 'wayfFilter' DESC 'WAYF filter for the facility' EQUALITY caseExactMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
+olcAttributeTypes: {33}( 1.3.6.1.4.1.8057.2.80.39 NAME 'wayfEFilter' DESC 'WAYF filter for the facility' EQUALITY caseExactMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
+olcAttributeTypes: {34}( 1.3.6.1.4.1.8057.2.80.40 NAME 'isTestSp' DESC 'is test SP?' EQUALITY caseExactMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
```

Comment:
This attribute is actually also boolean type in perun (it was wrongly defined in the issue). We can then map it to the boolean syntax in LDAP and not string.

Reply:
Not relevant anymore.